docs/source/configuration/index.rst

```diff
@@ -27,3 +27,13 @@ Robotidy reads and ignores paths from ``.gitignore`` and ``--exclude``. You can
 ``--extend-exclude`` with pattern::
 
     robotidy --extend-exclude skip_me.robot|some_dir/* .
+
+.. rubric:: Target Version
+
+Robotidy can automatically disable transformers that are not supported in target version of Robot Framework.
+Typical usage is when your environment have Robot Framework >5.0 installed but your source code supports only previous
+Robot Framework version::
+
+    robotidy --target-version -rf4 .
+
+It will disable all transformers that require Robot Framework >4.0 to run (even if you have Robot Framework >4.0 installed).
```

Comment:
So when you set target version, you disable all transformers that require this version of RF? Isn't it confusing? I would expect the opposite. Correct me if I'm wrong.

Reply:
@mnojek I read it as:

> It will disable all transformers that require Robot Framework greater that <target-version> to run (even if you have Robot Framework greater that <target-version> installed).

Reply:
@Ga22be Exactly this. I think you worded it better so I will use your version instead :)